modules/nci/src/popup.cpp

```diff
@@ -98,6 +98,7 @@ void Popup::run()
             case 'q':
                 end_execution = true;
                 wclear(_border);
+                nci::IManager::redraw();
```

Comment:
Remove nci namespace if not needed.

Reply:
Indeed, I removed  nci namespace from line 17.